data.js

```diff
@@ -42,21 +42,6 @@ module.exports = [
       'UI/UX',
     ],
   },
-  {
```

Comment:
@spences10 I think you accidentally did a **cut & paste** instead of a **copy & paste** here 😄

Reply:
I saw that and thought that I needed to revise my branch